lib/ansible/module_utils/basic.py

```diff
@@ -2,6 +2,8 @@
 # Copyright (c), Toshio Kuratomi <tkuratomi@ansible.com> 2016
 # Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
 
+from __future__ import absolute_import, division, print_function
```

Comment:
You always have to check whether these are safe to add when you do.  I took a look and print_function looks safe (We use parenthesis appropriately for every case).  division is safe but when we add it we should change line 620 as well:
``` python
-return '%.2f %s' % (float(size) / limit, suffix)
+ return '%.2f %s' % (size / limit, suffix)
```